### ISSUE TITLE
bindWithDispatch arguments order

### ISSUE THREAD

**Issue by bertho-zero** [feathers-plus/feathers-redux#41]
In the "Autobind Action Creators" part of the readme I see:
```
const services = bindWithDispatch(store.dispatch, rawServices)
```

The signature of the redux binder is `bindActionCreators(actionCreators, dispatch)`, so I propose to respect the order of the arguments of the redux doc, ie:
```
const services = bindWithDispatch(rawServices, store.dispatch)
```

**Comment by eddyystop** [comment]
Wouldn't this be a breaking change?

**Comment by subodhpareek18** [comment]
Yes, this would be a breaking change, my bad I didn't realize this while adding the method.

**Comment by eddyystop** [comment]
You may be right theoretically but I'm not sure its worth the impact of a breaking change.

I'm putting a `won't fix` label on this as a reminder in case a big refactoring is done someday.
